flight/Modules/UAVOMSPBridge/UAVOMSPBridge.c

```diff
@@ -283,12 +284,13 @@ static void _msp_send_analog(struct msp_bridge *m)
 		FlightBatterySettingsGet(&batSettings);
 	}
 
-	// TODO:  Verify these with a board that has working power stuff
 	if (batSettings.VoltagePin != FLIGHTBATTERYSETTINGS_VOLTAGEPIN_NONE)
 		data.status.vbat = (uint8_t)lroundf(batState.Voltage * 10);
 
-	if (batSettings.CurrentPin != FLIGHTBATTERYSETTINGS_CURRENTPIN_NONE)
-		data.status.current = lroundf(batState.Current * 10);
+	if (batSettings.CurrentPin != FLIGHTBATTERYSETTINGS_CURRENTPIN_NONE) {
+		data.status.current = lroundf(batState.Current * 100);
```

Comment:
I'm not sure about this one.  It's just a protocol field local to this function and it's changed meaning between a couple versions (and is configurable-ish on the other end).  The protocol docs might be helpful, but it's not well documented there (in particular, the units are documented for voltage, but not current).  It used to be 1/10 of an amp, but at some point some systems changed it to 1/100th of an amp.  Default baseflight is 1/100th, so really *100 is just right for what worked in my out of the box config.